README.md

```diff
@@ -283,6 +301,37 @@ Creating organization 'abcMega Corporation'... done
 | Limit | Usage limit |   | x | x | x |   |
 | Description | Host collection description |   | x | x | x |   |
 
+## Products
+
+| Additional arguments | Description |
+| ---------------:| :--------------|
+| --organization | Only process organization matching this name |
+| --no-sync | Do not attempt to sync repositories |
+
+**Overview**
+* Due to the length of time that syncing repositories can take, the --no-sync option may be used to skip this step. To always disable syncing, ':products_no_sync: true' may be specified in configuration file.
```

Comment:
How about an option titled `sync_products` instead? By default, it's assumed to be `true`, and then the user sets it to `false` if they want to override the default. Having words like `no` or `disable` in options that are boolean has always seemed really confusing to me.